tox.ini

```diff
@@ -29,6 +30,32 @@ deps =
     subprocess: subprocess32
 whitelist_externals = open
 
+[testenv:integration-tests]
+deps =
+    tox
+commands =
+    tox -e pypy-requests -- -k "tests/test_integration.py" {posargs:-vvs}
+    tox -e py27-requests -- -k "tests/test_integration.py" {posargs:-vvs}
+    tox -e py36-requests -- -k "tests/test_integration.py" {posargs:-vvs}
```

Comment:
this aggregate env should probably just call `tox -e pypy-integration` / `tox -e py27-integration` / `tox -e py36-integration` now that those exist?

(but probably in the order of 27, 36, pypy?)

Reply:
Will do.